CHANGELOG.md

```diff
@@ -1,4 +1,16 @@
-## dbt 0.16.next (Release TBD)
+## dbt next (release TBD)
+### Features
+- Added --fail-fast argument for dbt run and dbt test to fail on first test failure or runtime error. ([#1649](https://github.com/fishtown-analytics/dbt/issues/1649))
```

Comment:
Please add a link to this PR as well, and add a new contributors section for this version that has you + this PR!